### DISCUSSION TITLE
How to disable selective features?

### DISCUSSION THREAD

**Discussion by Praful** [marlonrichert/zsh-autocomplete#604] [Q&A]
Is it possible to switch off the following features selectively:

- the history popup that appears when I press K (in Vi mode)? I use fzf for history. (As an aside, I find the history unusable in Vi mode. I can't exit the history (I'd expect ESC to exit) and constant semi-colons appear and the line is not what it appears to be eg `ls abcb;` turns into `ls abc` when I press enter.)

- all keyboard bindings

The most useful feature I find is the appearance of help eg for commands switches over and above the normal zsh functionality. Could I disable everything apart from that?

I've disabled this plugin for now because of issue https://github.com/marlonrichert/zsh-autocomplete/issues/594.

**Comment by marlonrichert** [comment]
Add to your `.zshrc` file _before_ sourcing Autocomplete:
```zsh
zstyle ':autocomplete:key-bindings' enabled no
```
